datastore/save.go

```diff
@@ -53,6 +54,32 @@ func reflectFieldSave(props *[]Property, p Property, name string, opts saveOpts,
 	switch x := v.Interface().(type) {
 	case *Key, time.Time, GeoPoint:
 		p.Value = x
+	case civil.Date, civil.DateTime, civil.Time:
+		if dateVal, ok := v.Interface().(civil.Date); ok {
+			p.Value = dateVal.In(time.Now().Location())
+			*props = append(*props, p)
+			return nil
+		}
+		if timeVal, ok := v.Interface().(civil.Time); ok {
+			var format string
```

Comment:
Is there a particular reason to use `time.Time` and try to figure out formatting, rather than using `civil.Date.MarshalText` and `civil.Date.UnmarshalText`?

Reply:
If we'll use Marshal*, we'll get just time "hh:mm:ss". With current implementation we're getting "yyyy-mm-dd hh:mm:ss +tz". I wasn't very sure which variant is more correct. I'm okay to use Marshal* methods, if it's okay.